CMR-6738 (#1070)
nasa/Common-Metadata-Repository: search-app/docs/api.md

When all the results have been returned subsequent calls using the same `CMR-Scroll-Id` header will return an empty list.

> Important note: Clients use scrolling (especially via programatic api or scripts) should explicitly invoke [`clear scroll session`] (#clear-scroll) to release the scroll session when they are done. This will allow the scroll sessions be released promptly to free up system resources.

Reviewer: Grammar/style nitpick:
```suggestion
Important note: Clients using scrolling (especially via programatic api or scripts) should explicitly invoke [`clear scroll session`] (#clear-scroll) to release the scroll session when they are finished. This will end the scroll session and free up system resources.
```

    curl -i -XPOST -H "Content-Type: application/json" %CMR-ENDPOINT%/clear-scroll -d '{ "scroll_id" : "xxxx"}'

> It returns HTTP status code 204 when successful.

Reviewer 2: We may not be returning anything, but isn't the post a command to do something? Would 200 be better?

Author: 204 is success without content.
